doc/Pacemaker_Explained/en-US/Ch-Constraints.txt

```diff
@@ -43,11 +42,13 @@ the score (possibly as part of a custom resource agent).
 
 indexterm:[Location Constraints]
 indexterm:[Resource,Constraints,Location]
-There are two alternative strategies for specifying which nodes a
-resources can run on.  One way is to say that by default they can run
-anywhere and then create location constraints for nodes that are not
-allowed.  The other option is to have nodes "opt-in" --  to start with
-nothing able to run anywhere and selectively enable allowed nodes.
+'Location constraints' tell the cluster which nodes a resource can run on.
+
+There are two alternative strategies. One way is to say that, by default,
+resources can run anywhere, and then the location constraints specify nodes
+that are not allowed (an 'opt-out' cluster). The other way is to start with
```

Comment:
How so? It already says above that the highest score wins, and the table immediately below describes how a score works for a location constraint. Maybe some examples in the detailed sections below?